core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksBlockMetaStore.java

```diff
@@ -275,15 +276,17 @@ public Optional<BlockMeta> getBlock(long id) {
     try {
       meta = db().get(mBlockMetaColumn.get(), Longs.toByteArray(id));
     } catch (RocksDBException e) {
-      throw new RuntimeException(e);
+      ProcessUtils.fatalError(LOG, e, "RocksDB failed");
```

Comment:
In the current implementation before this change, if a RocksDB block request fails, a RuntimeException will be thrown and only the request will fail. If the Master RocksDB fails, then all block related requests will fail without crashing the master. 

There's a tricky difference for the RocksInodeStore though. Although the RocksInodeStore implementation is the same by throwing RuntimeException but not crashing the master, the InodeTreePersistentState will crash the master on seeing the RuntimeException. 

So the current behavior is:
1. RocksDB fails a block request -> only the RPC will fail
2. RocksDB fails an inode request -> RPC will fail and master will crash

So even without this change, even if the block req doesn't crash the master, the next inode req will. I really can't say for sure what is the best thing to do, but I figure we should keep the behavior consistent between the block and inode store, and the easiest way to do so is to crash the same way.

Reply:
@tcrain PTAL if this makes sense to you. Thx!